src/sentry/static/sentry/app/views/settings/team/teamMembers.jsx

```diff
@@ -173,27 +177,37 @@ const TeamMembers = createReactClass({
       .filter(m => !existingMembers.has(m.id))
       .map(m => {
         return {
-          value: m.id,
-          label: m.name || m.email,
+          value: `${m.name} ${m.email}`,
+          memberId: m.id,
+          label: <StyledUserListElement user={m} />,
         };
       });
 
+    let dropdownLabel = (
+      <StyledMembersLabel>
+        Members
+        <StyledCreateMemberLink
+          to={`/settings/organization/${params.orgId}/members/new/`}
+        >
+          + Add Member
+        </StyledCreateMemberLink>
+      </StyledMembersLabel>
+    );
+
+    let group = {
```

Comment:
Will this be displayed as a search result and/or get filtered when searching?

Reply:
It isn't a searchable item, which I think is correct, but when you search for something with no results, DropdownAutocomplete removes it (this is a feature for multiple groups) which is bad since that might be the time when a user decided to click "add new". 

We could add a `dontHide` key in the data or something or we could just stick the label in as part of `children`. First thing is easier in the short term.

![add-member-3](https://user-images.githubusercontent.com/435981/37367493-c2fe2f8e-26c0-11e8-9b56-7881ada3b303.gif)

Reply:
(Non-blocking for this PR)

Ah I see.

I think groups complicates the search a bit, what about instead of the `action` prop we currently have, it becomes `menuHeader` + `menuFooter`, and this just becomes a header.

Here's an example where this is helpful:
When you search and there are no results (but you have a group always show), `this.autoCompleteFilter` will return an array w/ length == 1 (the group header). If you want to include an empty state, you'd have to now include logic to get the count of items w/o groups. Of course we may need to go this route anyway if we have more components w/ multiple groups.

Reply:
that's fine. I'll definitely wait to implement this tho, because we'll probably want to extract the styles and structure for the labels into something.